### PR TITLE
Sidekiq scheduler

### PR DESCRIPTION
**Pull request by camilo9020**

### CODE REVIEW

**Comment by camilo9020** [comment]
[[4p/13v] Deprecar `clock`, migrar a sidekiq y usar un thread de `sidekiq` como scheduler.](https://trello.com/c/14xsgalB/431-4p-13v-deprecar-clock-migrar-a-sidekiq-y-usar-un-thread-de-sidekiq-como-scheduler)

**Comment by blackjid** [comment]
Hice una prueba y nunca me pregunto por el schedule

```
Which database are you using?: PostgreSQL
Do you want to use Devise for authentication? (required for ActiveAdmin): No
Which front-end framework are you going to use?: Vue
Which email service are you using?: None, thanks
Do you want to use Sidekiq for background job processing?: Yes
Do you need to report application errors with Sentry?: Yes
What is the main locale of your app?: es-CL (Chile)
Do you want to enable API support?: No
Do you want to use Draper to decorate models?: No
Do you want to use Paperclip for uploads?: No
Are you going to deploy to heroku? (contacto@platan.us): No
Do you want to create the Github repository (https://github.com/platanus/test-side-sch) for this project?: No
```

File: lib/potassium/recipes/background_processor.rb [hunk 1]
```diff
@@ -1,22 +1,18 @@
...
-
-    set(:background_processor, response.to_sym)
+    response = if selected?(:email_service, :none)
+                answer(:background_processor) do
+                  Ask.confirm("Do you want to use Sidekiq for background job processing?")
+                end
+              else
+                info "Note: Emails should be sent on background jobs. We'll install sidekiq"
+                true
+              end
```
Comment (monkeyci) [inline lib/potassium/recipes/background_processor.rb]: end at 10, 14 is not aligned with if at 3, 15.

File: lib/potassium/recipes/background_processor.rb [hunk 2]
```diff
@@ -1,22 +1,18 @@
...
-    response = answer(:background_processor) do
-      options.keys[Ask.list("Which background processor are you using?", options.values)]
-    end
-
-    set(:background_processor, response.to_sym)
+    response = if selected?(:email_service, :none)
+                answer(:background_processor) do
+                  Ask.confirm("Do you want to use Sidekiq for background job processing?")
+                end
+              else
```
Comment (monkeyci) [inline lib/potassium/recipes/background_processor.rb]: Align else with if.

File: lib/potassium/recipes/background_processor.rb [hunk 3]
```diff
@@ -1,22 +1,18 @@
...
-      none: "None, thanks"
-    }
-
-    response = answer(:background_processor) do
-      options.keys[Ask.list("Which background processor are you using?", options.values)]
-    end
-
-    set(:background_processor, response.to_sym)
+    response = if selected?(:email_service, :none)
+                answer(:background_processor) do
```
Comment (monkeyci) [inline lib/potassium/recipes/background_processor.rb]: Use 2 (not 1) spaces for indentation.

Comment (blackjid) [inline lib/potassium/recipes/background_processor.rb]: yapo felipe!

**Pull request merged**